overlord/hookstate/ctlcmd/refresh_test.go

```diff
@@ -204,6 +204,23 @@ version: 1
 	gating = nil
 	c.Assert(s.st.Get("snaps-hold", &gating), IsNil)
 	c.Check(gating["foo"]["snap1"], IsNil)
+
+	mockContext.Cache("action", nil)
+
+	mockContext.Unlock()
+	defer mockContext.Lock()
+
+	// refresh --pending --proceed is the same as just saying --proceed.
+	stdout, stderr, err = ctlcmd.Run(mockContext, []string{"refresh", "--proceed"}, 0)
```

Comment:
Don't you need to add `"--pending"` here? Otherwise, this block you added seems to me to be exactly the same as the block above.

Reply:
Oh, of course, thanks for catching! /o\